tests/kuttl/tests/collapsed/01-assert-collapsed-cell.yaml

```diff
@@ -0,0 +1,112 @@
+apiVersion: core.openstack.org/v1beta1
+kind: OpenStackControlPlane
+metadata:
+  name: openstack-collapsed-cell
+spec:
+  secret: osp-secret
+  keystone:
+    template:
+      databaseInstance: openstack
+      secret: osp-secret
+  mariadb:
+    template:
+      storageRequest: 500M
```

Comment:
The `core_v1beta1_openstackcontrolplane_collapsed_cell.yaml` sample does not have a `template` for `mariadb`, so this will not match.